recipes/libultrahdr/all/conanfile.py

```diff
@@ -0,0 +1,102 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir
+
+import os
+
+required_conan_version = ">=2.0.0"
+
+
+class LibultrahdrConan(ConanFile):
+    name = "libultrahdr"
+    description = "libultrahdr is an image format for storing SDR and HDR versions of an image for android."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/google/libultrahdr"
+    license = "Apache-2.0"
+    topics = ("ultrahdr", "graphics", "image")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_jpeg": [False, "libjpeg", "libjpeg-turbo", "mozjpeg"],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_jpeg": "libjpeg",
+    }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        if self.options.with_jpeg == "libjpeg":
+            self.requires("libjpeg/9e")
+        elif self.options.with_jpeg == "libjpeg-turbo":
+            self.requires("libjpeg-turbo/3.0.0")
+        elif self.options.with_jpeg == "mozjpeg":
+            self.requires("mozjpeg/4.1.3")
+
+    def validate(self):
+        check_min_cppstd(self, 17)
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
```

Comment:
```suggestion
    def source(self):
        get(self, **self.conan_data["sources"][self.version], strip_root=True)
        apply_conandata_patches(self)
```

(no need for a `_patch_sources()` method either, unless it has more complex logic)

Reply:
thanks!